debian/rules

```diff
@@ -166,8 +179,9 @@ override_dh_install:
 		>debian/tmp/var/lib/delphix-appliance/platform
 	rm debian/tmp/var/lib/delphix-appliance/platform.in
 
-	./scripts/download-signature-key.sh upgrade 5.3 \
-		>debian/tmp/var/lib/delphix-appliance/key-public.pem.upgrade.5.3
+	for keyType in $(SIGN_KEY_TYPES) ; do \
```

Comment:
we use [snake case](https://en.wikipedia.org/wiki/Snake_case) already in this file, so it feels inconsistent to use [camel case](https://en.wikipedia.org/wiki/Camel_case) for this one variable. I recommend we change `keyType` to `type` and avoid this issue entirely (IMO, we don't lose anything in terms of readability by doing this).

Reply:
Sounds good, will rename to just `type`.

Reply:
Fixed.